index.html

```diff
@@ -1230,9 +1228,9 @@ <h3>Avoid correlation</h3>
       </p>
 
       <p>
-        A DID method implementer should rotate keys and identifiers as often as
-        possible
-        to avoid correlation
+        DID method implementations are encouraged to rotate keys and
+        identifiers as often as possible,
+        to avoid correlation.
```

Comment:
```suggestion
        identifiers as often as practical,
        to avoid compromise and correlation, respectively.
```

Reply:
Applied in 45d8529cbc7f5667d5038475e90643f9fe9bad44.

Related earlier discussion: https://github.com/w3c/did-imp-guide/pull/37#discussion_r759440231